app/src/App.tsx

```diff
@@ -147,7 +168,11 @@ export default function App() {
       try {
         const initialUrl = await Linking.getInitialURL();
 
-        if (Platform.OS !== 'web' && initialUrl == null) {
+        if (
+          Platform.OS !== 'web' &&
+          Platform.OS !== 'macos' &&
+          initialUrl == null
+        ) {
```

Comment:
I don't mind using `Platform.OS` so in example app code, but what do you think about the following?

```ts
const IS_WEB = Platform.OS === 'web';
const IS_MACOS = Platform.OS === 'macos';
```

```suggestion
        if (!IS_WEB && !IS_MACOS && initialUrl == null) {
```